src/Workload/Microsoft.NET.Sdk.Maui/WorkloadManifest.in.json

```diff
@@ -282,9 +284,19 @@
       "kind": "library",
       "version": "@MAUI_GRAPHICS_VERSION@"
     },
-    "Microsoft.Maui.Sdk": {
+    "Microsoft.Maui.Sdk-@MAUI_DOTNET_VERSION@": {
```

Comment:
I think you could keep this one `Microsoft.Maui.Sdk` without a version.

Reply:
If I do that, then I get issues because the Sdk-7.0 is an alias to this one. This results in the autoimports not working for net6 if I recall. Android works because they have alias-to, but in maui we will have a mix and a self dependency. It ignores the version on the alias-to if the alias is to a pack in the workload.

Reply:
Changing `"kind": "sdk"` to `"framework"` should be what would stop an `AutoImport.props` from working. I only did that in the Android workload temporarily, until we shipped service release, such as:

https://github.com/xamarin/xamarin-android/commit/2ca2a103d4e473eeeceecd7689961038bb374cdb

Reply:
I see. I tried several combinations of things and I guess the tests that I checked with net6 coincidentally all had framework :)